public/scss/modules/_photo.scss

```diff
@@ -37,6 +37,9 @@ Photo
 }
 
 .thumbnail .caption {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
```

Comment:
Do we want the album names to be shortened with an ellipsis, or do we want to make the row distance variable?

Reply:
well it stimulates people to come up with short album names

Reply:
Agreed. Merge it I'd say. Can you do this @jszanto ?